utils/resource/usage.go

```diff
@@ -84,6 +88,11 @@ func NewManager(diskPath string, frequency, cpuHalflife, diskHalflife time.Durat
 		onClose:            make(chan struct{}),
 		availableDiskBytes: math.MaxUint64,
 	}
+
+	metricsRegisterer := prometheus.NewRegistry()
+	processMetrics, _ := newMetrics("System resources", metricsRegisterer, m.processes)
```

Comment:
`m.processes` is always empty here - so I don't think we need to pass it into `newMetrics`, we can just leave the metric values as uninitialized.

Reply:
updated